Sources/StreamChatUI/StreamSwiftyMarkdown/SwiftyMarkdown/PerfomanceLog.swift

```diff
@@ -16,7 +16,7 @@ class PerformanceLog {
 	
 	init( with environmentVariableName : String, identifier : String, log : OSLog  ) {
 		self.log = log
-		self.enablePerfomanceLog = (ProcessInfo.processInfo.environment[environmentVariableName] != nil)
```

Comment:
why do we need to change this? It's in the markdown library. If really needed, probably we need to update something in the scripts, since the change would be overwritten if we re-embed it.

Reply:
`ProcessInfo.processInfo.environment` keeps showing up in the profile (~15ms). Since it is not used, I ended up disabling it.

Reply:
ok, please check with Alexey if changing it only here is enough.

Reply:
We need to delete `SwiftyMarkdown` from the `Makefile`. We've already implemented a ton of breaking changes. We are not going to update it ever.

Reply:
Sure, I can add that although I am sure we are not going to update it (it looks abandoned)

Reply:
@testableapple done